server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/subsystem/CacheContainerConfigurationBuilder.java

```diff
@@ -226,6 +227,9 @@ public GlobalConfiguration getValue() {
                 .allowDuplicateDomains(true);
 
         builder.addModule(PrivateGlobalConfigurationBuilder.class).serverMode(true);
+
+        // ISPN-7644 Set the default cache name so that ISPN000435 warning is not displayed on server startup
+        builder.defaultCacheName(CacheContainer.DEFAULT_CACHE_NAME);
```

Comment:
Actually, the server shouldn't even bother with the default cache. If it is starting it, then we need to prevent that.